### ISSUE TITLE
Calling non-static methods statically

### ISSUE THREAD

**Issue by cw-rahulwankhade** [spatie/laravel-analytics#83]
In documentation, methods are called statically while they are non static and giving error "non static method can not be called statically".
e.g. Analytics::fetchMostVisitedPages(Period::days(7)); 

If I create the object of class and then call the method then I have to pass the instance of other class objects.
 e.g while creating object of Analytics class I have to pass the instance of AnalyticsClient class. Also for creation of AnalyticsClient class object, I have to pass Google_Service_Analytics class instance which  subsequently requires the instance of Google_Client. 

But from where to obtain class definition of Google_Client and Google_Service_Analytics.

In short, How can I call these methods from my controllers in order to obtain google analytics data?

**Comment by sebastiandedeyne** [comment]
The example in the docs is using the Laravel facade class. In your controller you should `use Analytics;` instead of importing the real class.

https://laravel.com/docs/master/facades
